tests/pages/donate.py

```diff
@@ -0,0 +1,10 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pages.page import Page
+
+
+class DonatePage(Page):
+
+    _url = 'https://donate.mozilla.org/'
```

Comment:
While I understand why this is here, it seems a bit odd to have a page object for something that's not part of our site. Maybe this is the easiest solution though?